kivy/uix/widget.py

```diff
@@ -261,7 +261,8 @@ def _widget_destructor(uid, r):
     # Internal method called when a widget is deleted from memory. the only
     # thing we remember about it is its uid. Clear all the associated callbacks
     # created in kv language.
-    del _widget_destructors[uid]
+    if uid in _widget_destructors:
```

Comment:
You can do this in one line: `_widget_destructors.pop(uid, None)`

Reply:
@LESSSE pls do the requested changes so that PR can be merged

Reply:
Done, Thanks